### DISCUSSION TITLE
check the activated clients on docker-kms

### DISCUSSION THREAD

**Discussion by shqrif** [11notes/docker-kms#5] [General]
Hi , 

I want to thank you for the impressive docker of kms which you create .. I want to ask how I can know the clients or get a list of them from the docker  ... Thank you very much . I saw a WHO DB ... So how i can get it

**Comment by 11notes** [comment]
Don't thank me, thank the developers at https://github.com/Py-KMS-Organization/py-kms.

> I want to ask how I can know the clients or get a list of them from the docker

You can simply open the sqlite database. WhoDB is just an example on how to do that.

**Comment by 11notes** [comment]
@shqrif there is no a web gui at: https://github.com/11notes/docker-kms-gui
